pkg/dazzle/project.go

```diff
@@ -330,7 +331,7 @@ func loadChunks(dir fs.FS, contextBase, base, name string) (res []ProjectChunk,
 		} else if err != nil {
 			return nil, fmt.Errorf("%s: cannot read tests.yaml: %w", dir, err)
 		}
-		err = yaml.UnmarshalStrict(tf, &chk.Tests)
+		err = yaml.Unmarshal(tf, &chk.Tests)
```

Comment:
To preserve [strict mode behavior](https://ubuntu.com/blog/api-v3-of-the-yaml-package-for-go-is-available#:~:text=Strict%20mode%20is%20now%20Decoder.KnownFields), should we do something like this on L345?

```go
decoder := yaml.NewDecoder(fd)
decoder.KnownFields(true)
err = decoder.Decode(&cfg)
```

Reply:
Good point. Fixed.